mitmproxy/tools/console/common.py

```diff
@@ -266,9 +266,95 @@ def colorize_url(url):
         ('url_punctuation', 3),  # ://
     ] + colorize_host(parts[2]) + colorize_req('/' + parts[3])
 
+@lru_cache(maxsize=800)
+def raw_format_list(f):
```

Comment:
What is the rationale for adding this? Why not make the option use the two-row layout from the tabular view always?